src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessDailyListAutomatedTask.java

```diff
@@ -14,20 +17,18 @@
 import static uk.gov.hmcts.darts.task.api.AutomatedTaskName.PROCESS_DAILY_LIST_TASK_NAME;
 
 @Slf4j
+@Component
 @SuppressWarnings({"squid:S1135"})
-public class ProcessDailyListAutomatedTask extends AbstractLockableAutomatedTask {
+public class ProcessDailyListAutomatedTask
+    extends AbstractLockableAutomatedTask
+    implements AutoloadingManualTask {
 
     protected String taskName = PROCESS_DAILY_LIST_TASK_NAME.getTaskName();
     private DailyListProcessor dailyListProcessor;
```

Comment:
Suggestion: How about making this `final` as well?

Reply:
Agreed, will update apart of the AutomationTask update ticket